demo/src/screens/ThankScreen/ThankScreen.js

```diff
@@ -1,13 +1,88 @@
-import React from 'react';
+import React, {useMemo} from 'react';
+import {Container, Row, Col, Jumbotron} from 'react-bootstrap';
+import {ProductListing} from '../../components/ProductListing/ProductListing.js';
+import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
 
 /**
+ * @param {!Object} items The items stored in the site state
```

Comment:
Should be 
`@param {{items: Array}} items ...`

per https://google.github.io/styleguide/jsguide.html#features-objects-destructuring


Also, it's practice to end descriptions with a dot.